README.md

```diff
@@ -46,21 +46,36 @@ pip install appmap
 For projects that use `poetry` , add the `appmap` package to `pyproject.toml`.
 
 ## Configuration
-Add your modules as `path` entries in `appmap.yml`:
+Add your modules as `path` entries in `appmap.yml`, and external packages
+(distributions) as `dist`:
 
 ```yaml
 name: my_python_app
 packages:
 - path: app.mod1
+  shallow: true
 - path: app.mod2
   exclude:
   - MyClass
   - MyOtherClass.my_instance_method
   - MyOtherClass.my_class_method
+- dist: Django
+  exclude:
+  - django.db
 ```
 
 Note that `exclude`s are resolved relative to the associated path. So, for example, this
-configuration excludes `app.mod2.MyClass`
+configuration excludes `app.mod2.MyClass`.
+
+`dist` specifiers use importlib.metadata, and so should work regardless of
+the package flavor (egg, wheel, etc.). You can additionally use `path` and `exclude`
+on `dist` modules to limit the capture to specific patterns.
```

Comment:
This should be "`dist` packages", no?

Reply:
I suppose... I think I'll change it to "entries".